client/src/services/notificationManager.ts

```diff
@@ -67,11 +67,9 @@ export class NotificationManager {
 
     const alertModalConfig: MsAlertModalConfig = {
       theme: this._getMsReportTheme(notification.level),
+      title: this.t(notification.title),
```

Comment:
Translation should be done by the caller, not here.

Reply:
Updated